InvenTree/InvenTree/status_codes.py

```diff
@@ -122,36 +116,44 @@ class PurchaseOrderStatus(StatusCode):
     ]
 
     # Failed orders
-    FAILED = [
-        CANCELLED,
-        LOST,
-        RETURNED
-    ]
+    FAILED = [CANCELLED, LOST, RETURNED]
```

Comment:
In general PEPs recommend to write list items one item per line to make diffs easier readable

Reply:
Ok, no problem